### PR TITLE
ext auth support failure mode allow header add config

### PR DESCRIPTION
**Pull request by StarryVae**
Commit Message: ext auth support failure mode allow header add config
Additional Description: N/A
Risk Level: low
Testing: ut
Docs Changes: N/A
Release Notes: N/A

### CODE REVIEW

**Comment by StarryVae** [comment]
issue #28387

File: changelogs/current.yaml [hunk 1]
```diff
@@ -127,6 +127,10 @@ minor_behavior_changes:
 - area: router
   change: |
     Enable environment_variable in router direct response.
+- area: ext_authz
+  change: |
+    Ext_authz supports adding ``x-envoy-auth-failure-mode-allowed`` header optionally when ``failure_mode_allow`` is enable,
+    and runtime flag ``envoy.reloadable_features.http_ext_auth_failure_mode_allow_header_add`` is removed.
```
Comment (mattklein123) [inline changelogs/current.yaml:133]: This should go into the removed runtime section.

**Review by mattklein123 — CHANGES_REQUESTED** [review]
Makes sense with small comment, thanks.

/wait

File: changelogs/current.yaml (hunk 1 above)
Comment (StarryVae) [reply changelogs/current.yaml]: done, PTAL again, thanks.

**Review by mattklein123 — APPROVED** [review]
Thanks!

**Pull request merged**